doc/builders/packages/linux.section.md

```diff
@@ -39,3 +42,33 @@ How to add a new (major) version of the Linux kernel to Nixpkgs:
 4.  Test building the kernel: `nix-build -A linuxKernel.kernels.kernel_2_6_22`. If it compiles, ship it! For extra credit, try booting NixOS with it.
 
 5.  It may be that the new kernel requires updating the external kernel modules and kernel-dependent packages listed in the `linuxPackagesFor` function in `linux-kernels.nix` (such as the NVIDIA drivers, AUFS, etc.). If the updated packages aren’t backwards compatible with older kernels, you may need to keep the older versions around.
```

Comment:
Do I need to do anything more than create `pkgs/os-specific/linux/kernel/README.md` with that information, or is this enough?

Reply:
That's enough. A slight issue may be that it won't be linked from anywhere, so one pinned GitHub URL from this section may be worthwhile. Not critical though IMO.

Reply:
I'm afraid we might have to wait for the PR to be merged before we add a pinned GitHub URL, because this PR is the one that generates the new `README.md` file :smiling_face_with_tear:

Reply:
Hm, you could have a commit for the README and then one with the link. The commit hash is what ends up in the URL if I'm not completely mistaken.